test/e2e/specs/basic.disabled_e2e.js

```diff
@@ -0,0 +1,50 @@
+import TestAppPage from '../page-objects/test-app.page';
```

Comment:
Test disabled as regex `*.e2e.js` in `wdio.conf.js` won't pick this spec. 
This test doesn't work on OIE orgs (Current test org is OIE)
Retaining the test for historical context if we need to enable this test later in OIE.